rs/bitcoin/ckbtc/minter/src/state.rs

```diff
@@ -883,7 +893,7 @@ impl CkBtcMinterState {
         }
         self.tokens_burned += request.amount;
         if let Some(kyt_provider) = request.kyt_provider {
-            *self.owed_kyt_amount.entry(kyt_provider).or_insert(0) += self.kyt_fee;
+            *self.owed_kyt_amount.entry(kyt_provider).or_insert(0) += self.check_fee;
```

Comment:
KYT providers collect the KYT fee and not the check fee, right?

Reply:
Agree that it is a bit awkward. But we are going to remove the whole reimbursement and owed_kyt_amount handling after upgrading the ckBTC minter once (so that we can be sure all fees are distributed).